### PR TITLE
[IMP] fieldservice_helpdesk_stock: Add FSM Location to SRO's + SR's

### PR DESCRIPTION
**Pull request by osi-scampbell**
Ticket #1122

This PR adds FSM Locations to SR's and SRO's. Instead of going through the context I elected to handle the login in the create() and the onchange().

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
@osi-scampbell Can you add a test to cover that scenario?

**Review by bodedra — APPROVED** [review]
LGTM :+1: 

@osi-scampbell It would be good if we make travis-ci green.

Firefox_Screenshot_2019-10-24T05-07-35 062Z

**Comment by osi-scampbell** [comment]
> LGTM 👍
> 
> @osi-scampbell It would be good if we make travis-ci green.
> 
> Firefox_Screenshot_2019-10-24T05-07-35 062Z

I totally forgot about the lint errors...thanks for pointing that out @bodedra, I'll take care of that now

File: fieldservice_helpdesk_stock/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
```
Comment (max3903) [inline fieldservice_helpdesk_stock/tests/__init__.py]:
```suggestion
# Copyright (C) 2019 Open Source Integrators
```

File: fieldservice_helpdesk_stock/tests/test_stock_request.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (max3903) [inline fieldservice_helpdesk_stock/tests/test_stock_request.py]:
```suggestion
# Copyright (C) 2019 Open Source Integrators
```

File: fieldservice_helpdesk_stock/views/stock_request.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (max3903) [inline fieldservice_helpdesk_stock/views/stock_request.xml]:
```suggestion
```

File: fieldservice_helpdesk_stock/views/stock_request_order.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (max3903) [inline fieldservice_helpdesk_stock/views/stock_request_order.xml]:
```suggestion
```

**Review by max3903 — CHANGES_REQUESTED** [review]
Few cosmetic changes.

**Comment by max3903** [comment]
@osi-scampbell Can you rebase this branch to get it green?

**Comment by osi-scampbell** [comment]
@max3903 Branch rebased, commits squashed

**Comment by bodedra** [comment]
Yay. It's :green_apple: now.

**Review by max3903 — APPROVED** [review]

**Pull request merged**